### PR TITLE
Release 1.5.0

### PR DESCRIPTION
**Pull request by bmjen**

### CODE REVIEW

File: locales/pdk.pot [hunk 1]
```diff
@@ -6,11 +6,11 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: puppet development kit v1.4.0-20-gdc7faf3\n"
+"Project-Id-Version: puppet development kit v1.4.1-76-ge120ca5\n"
```
Comment (bmjen) [inline locales/pdk.pot]: I've been informed by @eputnam that this version string doesn't matter.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,58 @@ All changes to this repo will be documented in this file.
...
+- \(MAINT\) Use `bundle lock --update` to pin json to built-in versions [\#460](https://github.com/puppetlabs/pdk/pull/460) ([scotje](https://github.com/scotje))
+
+**Closed issues:**
+
+- PDK should default to mock\_with :rspec and resolve deprecation message [\#477](https://github.com/puppetlabs/pdk/issues/477)
+- Support for template URL with a branch [\#447](https://github.com/puppetlabs/pdk/issues/447)
+- Any interest in adding a `--parallel` option to `pdk test unit`? [\#446](https://github.com/puppetlabs/pdk/issues/446)
+- Installing PDK from .deb causes unmet dependencies on Ubuntu 17.10 Artful [\#370](https://github.com/puppetlabs/pdk/issues/370)
+- Repo Configs Contain Invalid URLs [\#319](https://github.com/puppetlabs/pdk/issues/319)
+- Gems not found in pre-release [\#254](https://github.com/puppetlabs/pdk/issues/254)
```
Comment (scotje) [inline CHANGELOG.md]: Seems confusing to include this "closed issues" section in the Changelog, but it's what we've been doing so far I guess.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,58 @@ All changes to this repo will be documented in this file.
...
+- \(PDK-904\) Warns users of pdk version compatibility  [\#482](https://github.com/puppetlabs/pdk/pull/482) ([bmjen](https://github.com/bmjen))
+- \(maint\) Allow `pdk bundle` to work without `--` [\#466](https://github.com/puppetlabs/pdk/pull/466) ([DavidS](https://github.com/DavidS))
+- \(PDK-877\) Make PDK compatible with Ruby 2.5 [\#459](https://github.com/puppetlabs/pdk/pull/459) ([scotje](https://github.com/scotje))
+- Ruby 2.4.3 transition [\#453](https://github.com/puppetlabs/pdk/pull/453) ([bmjen](https://github.com/bmjen))
+- \(PDK-846\) add Resource API type unit test template [\#451](https://github.com/puppetlabs/pdk/pull/451) ([tphoney](https://github.com/tphoney))
+- \(PDK-785\) Add --puppet-version and --pe-version CLI options [\#448](https://github.com/puppetlabs/pdk/pull/448) ([rodjek](https://github.com/rodjek))
+
+**Fixed bugs:**
+
+-  undefined method `make\_tmpname' with  [\#455](https://github.com/puppetlabs/pdk/issues/455)
```
Comment (DavidS) [inline CHANGELOG.md]: huh?

Comment (DavidS) [reply CHANGELOG.md]: Fiexed up the issue title to be more descriptive

**Pull request merged**